contracts/Pool/PoolRegistry.sol

```diff
@@ -249,7 +249,7 @@ contract PoolRegistry is Ownable2StepUpgradeable {
             input.asset,
             comptroller,
             rate,
-            10**input.decimals,
+            10**18,
```

Comment:
VTokens should have 8 decimals. Shouldn't we just set the exchange rate in a way that when we deposit a token lets say with 18 decimals to result in 1e8 vTokens?

Reply:
In other words, exchange rate should be computed out of the decimal places of the underlying token... We should not assume that every token will have 18 decimal places right? 
We only assume that vTokens in isolated-pools will always have 8 decimals, but this is not the case for the underlying imo